src/python/pants/rules/core/fmt_test.py

```diff
@@ -80,7 +82,9 @@ def run_fmt_rule(self, *, targets: List[HydratedTargetWithOrigin]) -> Tuple[Fmt,
                 console,
                 HydratedTargetsWithOrigins(targets),
                 Workspace(self.scheduler),
-                UnionMembership({FormatTarget: [PythonTargetAdaptorWithOrigin]}),
+                UnionMembership(
+                    OrderedDict({FormatTarget: OrderedSet([PythonTargetAdaptorWithOrigin])})
+                ),
```

Comment:
This is a bummer to see because dicts are ordered in Python 3.6+, so using `OrderedDict` is redundant. But, that's outside the scope of this PR to switch to `dict` and Stu and Benjy weren't convinced we should because `OrderedDict` expresses the intent that you do care about the order. 

So, this is fine to change for now and worth it for all the new type hints we're getting :)